ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -44,6 +43,9 @@ def check(self, _):
             self.service_check(self.SERVICE_CHECK, AgentCheck.CRITICAL, self.config.tags)
             return
 
+        if self.is_metadata_collection_enabled():
+            self.set_metadata('version', self.queue_metric_collector._collect_metadata(queue_manager))
```

Comment:
I think it could be: 

```suggestion
            self.collect_metadata(queue_manager)
```

For consistency with other integrations.

You can then inside `self.collect_metadata` call `self.queue_metric_collector._collect_metadata`.